rest/results-tabulation-api/ext/ExtendedElection/ExtendedElectionParliamentaryElection2020/ExtendedTallySheet/ExtendedTallySheet_ALL_ISLAND_RESULT.py

```diff
@@ -0,0 +1,94 @@
+import math
+
+from flask import render_template
+
+from ext.ExtendedTallySheet import ExtendedTallySheet
+from orm.entities.Area import AreaModel
+from util import to_comma_seperated_num, to_percentage
+
+
+class ExtendedTallySheet_ALL_ISLAND_RESULT(ExtendedTallySheet):
```

Comment:
I'm doubt whether this is working. This should be ExtendedTallySheetReport

Reply:
This is fixed.